packages/@aws-cdk/aws-ecs/lib/fargate/fargate-service.ts

```diff
@@ -86,6 +94,9 @@ export class FargateService extends BaseService implements IFargateService {
       ...props,
       desiredCount: props.desiredCount !== undefined ? props.desiredCount : 1,
       launchType: LaunchType.FARGATE,
+      propagateTags: props.propagateTaskTagsFrom === undefined ? PropagatedTagSource.SERVICE :
```

Comment:
I feel like we can clean this logic up a bit with something like:

propagateTags: props.propagateTaskTagsFrom === undefined ? PropagatedTagSource.SERVICE : props.propagateTaskTagsFrom

And then add logic in the base service to handle undefined or PropagatedTagSource.NONE

Reply:
Agree